model/transformers/tests/test_ner_integration.py

```diff
@@ -0,0 +1,145 @@
+import csv
+import json
+import random
+import pathlib
+import contextlib
+
+from dffml.cli.cli import CLI
+from dffml.util.asynctestcase import IntegrationCLITestCase
+
+
+class TestNER(IntegrationCLITestCase):
+    async def test_run(self):
+        self.required_plugins("dffml-model-transformers")
+        TRAIN_DATA = []
+        PREDICT_DATA = []
+        DATA_LEN = 1
+        org_name = ["Tesla", "Facebook", "Nvidia", "Yahoo"]
+        per_name = ["Walter", "Jack", "Sophia", "Ava"]
+        loc_name = ["Germany", "India", "Australia", "Italy"]
+
+        sentences = []
+        for i in range(DATA_LEN):
+            sentences.append(
+                random.choice(per_name)
+                + " "
+                + random.choice(["works at", "joined", "left"])
+                + " "
+                + random.choice(org_name)
+                + " "
+                + random.choice(loc_name)
+            )
+
+        for id, sentence in enumerate(sentences):
+            PREDICT_DATA.append([id, sentence])
+
+        for id, sentence in enumerate(sentences):
+            example = []
+            words = sentence.split()
+            for word in words:
+                if word in org_name:
+                    example.append([id, word, "I-ORG"])
+                elif word in per_name:
+                    example.append([id, word, "I-PER"])
+                elif word in loc_name:
+                    example.append([id, word, "I-LOC"])
+                else:
+                    example.append([id, word, "O"])
+            TRAIN_DATA.extend(example)
+
+        train_data_filename = self.mktempfile() + ".csv"
+        predict_data_filename = (
+            train_data_filename.split(".")[0] + "predict.csv"
+        )
+
+        with open(pathlib.Path(train_data_filename), "w+") as data_file:
+            writer = csv.writer(data_file, delimiter=",")
+            writer.writerow(["A", "B", "target"])
+            writer.writerows(TRAIN_DATA)
+        with open(pathlib.Path(predict_data_filename), "w+") as data_file:
+            writer = csv.writer(data_file, delimiter=",")
+            writer.writerow(["A", "B"])
+            writer.writerows(PREDICT_DATA)
+
+        # Train the model
+        await CLI.cli(
+            "train",
+            "-model",
+            "ner_tagger",
+            "-model-SENTENCE_ID",
```

Comment:
```suggestion
            "-model-sid",
```

Let's also make this lower case